template/src/routes/Home/components/Home/Home.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import logo from './logo.svg';
```

Comment:
This original logo loading came straight from cra, is there a reason for referencing the file out of the statics folder instead? Is it because this was moved to a component?
We may want to call this `HomePage` instead of of just Home so that it is clear it is a route? Or maybe put it in a `routes` folder? Open to discussion on this

Reply:
The idea was to let the users know where the static files will be kept if they install and extend this template.
I agree with calling it `HomePage` instead of `Home`, so I will change it.